bin.src/createRefSpec.py

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env python
+#createRefSpec.py '/Users/azuri/spectra/pfs/PFS' --id visit=4 filter='r' spectrograph=2 site='F' category='A' -C ../obs_pfs/config/pfs/createRefSpec.py --output "/Users/azuri/spectra/pfs/PFS" --clobber-config
```

Comment:
This comment does seem to apply to this file, but it could do with a little context. Is this just an example of running the task that you might want to copy and paste? Is the content of the config override important?